scripts/py_matter_idl/matter_idl/tests/outputs/several_clusters/java/ClusterReadMapping.java

```diff
@@ -37,6 +37,7 @@ private static Map<String, InteractionInfo> readFirstInteractionInfo() {
           readFirstSomeIntegerCommandParams
         );
         result.put("readSomeIntegerAttribute", readFirstSomeIntegerAttributeInteractionInfo);
+      
```

Comment:
can you fix the templates to not add extra whitespace (it would be easier for humans to review code)

Reply:
Sure, I will fix the templates.